plugins/lvmprovisioner/provisioner.go

```diff
@@ -0,0 +1,133 @@
+package lvmprovisioner
+
+import (
+	"errors"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/gluster/glusterd2/pkg/fsutils"
+	"github.com/gluster/glusterd2/pkg/lvmutils"
+)
+
+var mountOpts = "rw,inode64,noatime,nouuid"
+
+// Provisioner represents lvm provisioner plugin
+type Provisioner struct{}
+
+func getVgName(devpath string) string {
```

Comment:
can we store the values like vgname, lvname, device name etc. As of now it return constant values, which is not the case with snapshot. 

Or is it possible to move the Mountinfo variable in the brickinfo to provisioner, and return this values based on the values stored

Reply:
I am trying to keep lvm specific things outside the device info. Non LVM use case will not have Vg. For example, loopmount devices provisioner